tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Documentation  Test 6-4 - Verify vic-machine create basic use cases
+Resource  ../../resources/Util.robot
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+*** Test Cases ***
+Create VCH - defaults
+    Log To Console  \nRunning vic-machine create - defaults
+    Set Test Environment Variables
+    Set Test VCH Name
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --image-datastore=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
+    Run Keyword And Ignore Error  Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}
+    Sleep  10 seconds
```

Comment:
Can you put the github tracker in here, so when/if that issue ever gets fixed we don't have to sleep for 30+ seconds for this test anymore?